bin/Ref_Maker

```diff
@@ -70,9 +71,10 @@ sub main { ##no critic (Subroutines::ProhibitExcessComplexity)
     print {*STDOUT} catfile( $CWD, 'fasta', $files[0] ), "\n" or carp $OS_ERROR;
     $ref_size = -s "fasta/$files[0]";
 
-    # If no arguments were supplied build all index files.
+    # If no arguments were supplied build all index files, except longranger
     if ( sum( values %build ) == 0 ) {
         foreach my $k ( keys %build ) { $build{$k} = 1; }
+	$build{longranger} = 0;
```

Comment:
Could we have single quotes aroud longranger to be consistent with the rest of the code?